caphandler/pluginStatus.go

```diff
@@ -70,3 +81,35 @@ func GetPluginStartup(ctx iris.Context) {
 	// TODO: implementation pending
 	ctx.StatusCode(http.StatusNotImplemented)
 }
+
+func publishResourceAddedEvent() {
```

Comment:
rename to publishFabricAddedEvent, as this is all the func does or will do in future

Reply:
updated